packages/spaces/lib/vpn-connections.js

```diff
@@ -9,6 +9,13 @@ module.exports = function (heroku) {
     })
   }
 
+  function getVPNConnection (space, nameOrUuid) {
+    return request('GET', `/spaces/${space}/vpn-connections/${nameOrUuid}`, {
+      space: space,
+      name_or_uuid: nameOrUuid
```

Comment:
Is this what the field is called? I think in @bernerdschaefer's PR for creating connections he just uses `name`. Standardizing on `name` makes sense to me and then just document that it can also take the UUID.

Reply:
@bernerdschaefer good call - I meant to clean this up to just `name`

Reply:
@watsonian good call - I meant to clean this up to just `name` 👍